src/views/profile.pug

```diff
@@ -57,3 +57,15 @@ block content
     span.icon
       i.fas.fa-shield-alt
     span= lang('PROFILE_SECURITY_CHANGE_PASSWORD')
+  | &nbsp;
+  if _CC.config.googleSSOEnabled
+    if  req.user.oauthConnections && req.user.oauthConnections.google
```

Comment:
There's an extra space at the beginning of this if statement.

Reply:
Resolved with https://github.com/Wingysam/Christmas-Community/pull/138/commits/599e27d877eb261d42b80de95c05c485bdf67333